stories/Button.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Button from 'ui/components/Button';
+
+const label = 'Hello Button';
+
+storiesOf('Button', module)
+  .add('default (no buttonType)', () => <Button>{label}</Button>)
```

Comment:
We can definitely iterate on this but at first glance, I don't think it will be useful to have one component style per page. I'd like to see many component styles on one page so that I can make a change to the code and see it all auto-update in front of me.

How about making one page per button type and showing each variation like this?

```diff
diff --git a/stories/Button.js b/stories/Button.js
index 0fa64a1c4..c61bf0796 100644
--- a/stories/Button.js
+++ b/stories/Button.js
@@ -6,6 +6,28 @@ import Button from 'ui/components/Button';
 const label = 'Hello Button';
 
 storiesOf('Button', module)
+  .add('action', () => {
+    return (
+      <dl className="stories-list">
+        <dt>Default</dt>
+        <dd>
+          <Button buttonType="action">{label}</Button>
+        </dd>
+        <dt>disabled=true</dt>
+        <dd>
+          <Button buttonType="action" disabled>{label}</Button>
+        </dd>
+        <dt>puffy=true</dt>
+        <dd>
+          <Button buttonType="action" puffy>{label}</Button>
+        </dd>
+        <dt>micro=true</dt>
+        <dd>
+          <Button buttonType="action" micro>{label}</Button>
+        </dd>
+      </dl>
+    );
+  })
   .add('default (no buttonType)', () => <Button>{label}</Button>)
   .add('default (no buttonType), disabled', () => (
     <Button disabled>{label}</Button>
diff --git a/stories/setup/App.override.styles.scss b/stories/setup/App.override.styles.scss
index 8b948128a..cdccdc5f1 100644
--- a/stories/setup/App.override.styles.scss
+++ b/stories/setup/App.override.styles.scss
@@ -1,6 +1,22 @@
+@import '~photon-colors/photon-colors';
+
 html,
 body {
   background: #fff;
   // Giving extra cushion so nothing gets covered.
   padding: 20px 10px 10px;
 }
+
+.stories-list {
+  display: grid;
+  grid-spacing: 0;
+  grid-template-columns: 1fr 1fr;
+
+  dd, dt {
+    align-items: center;
+    border-bottom: 1px solid $grey-30;
+    display: flex;
+    margin: 0;
+    padding: 6px;
+  }
+}
```

<img width="1098" alt="screenshot 2018-09-05 10 37 01" src="https://user-images.githubusercontent.com/55398/45104408-acf5ca00-b0f7-11e8-898b-231d1d95f627.png">

Reply:
yep, I like this. Before I was playing with knobs, I had something similar (well..all one way page) but I think this way is better. I will work on this..

Reply:
Instead of manually doing this, can we explore storybook addons? There are at least two addons for that:

- https://github.com/Checkfront/react-storybook-addon-chapters
- https://github.com/evgenykochetkov/react-storybook-addon-props-combinations

That's also why I initially requested a smaller patch with the bare minimum setup.. So that we can iterate over it, and actually ship things. FWIW, I have worked with Storybook in the past.

Reply:
ok so..I'll hold off on this piece for now? We can explore if chapters will do something similar in a follow up PR? will that work?

Reply:
> That's also why I initially requested a smaller patch with the bare minimum setup

I have been requesting changes because I didn't think the first pass was set up to meet our needs.

> Instead of manually doing this, can we explore storybook addons?

This sounds promising. To keep it moving, I suggest doing it in a follow-up issue.